.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,26 @@
+name: Continuous Integration
+on:
+  push:
+
+env:
+  RUBY_VERSION: '2.7.3'
+
+jobs:
+  rspec:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+      - run: bundle exec rake rspec
```

Comment:
Given the two jobs have the same set up I'd suggest running these as two steps in the same job, rather than separate jobs:


```
- run: bundle exec rake rspec
- run: bundle exec rake rubocop
```

Might also be worth setting the rubocop formatter to the builtin `github` one as that enables inline annotations.